### PR TITLE
Added CRAN downloads badge and link for Bug reports to DESCRIPTION

### PR DESCRIPTION
**Pull request by chrismainey**
- [ ] added unit tests and checked code coverage with `covr::report()` (should aim for 100%)
- [x] ran `devtools::document()`
- [ ] ran `lintr::lint_package()` and resolved all lint warnings and notes
- [ ] ran `styler::style_pkg()` to make sure code matches the style guidelines
- [x] ran R-CMD CHECK and resolved all issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,4 +120,4 @@ This is an NHS-R Community project that is open for anyone to contribute
 to in any way that they are able. If you want to learn more about this
 please join the discussion at [the NHS-R Community Slack
 group](https://nhsrcommunity.slack.com/) and the specific channel
-[#proj-shiny-spc](https://nhsrcommunity.slack.com/archives/CSVD4SYF3).
+[\#proj-shiny-spc](https://nhsrcommunity.slack.com/archives/CSVD4SYF3).
```
Comment (tomjemmett) [inline README.md:123]: I can't see that this was changed in the .Rmd, so not sure why it's flagging here... but, as the channel has changed to #proj-nhsr-plot-the-dots, could you add this change to the PR?

**Review by tomjemmett — COMMENTED** [review]
see slight minor change, but we could add that as another PR if you are pushed for time

File: README.md (hunk 1 above)
Comment (chrismainey) [reply README.md:123]: I don't know why it's done that.  The \ is not there in the README.Rmd, just in the knitted README.md.  Odd.

Comment (chrismainey) [reply README.md]: Added the new channel name though, and the link seems to work.

**Review by tomjemmett — APPROVED** [review]

**Pull request merged**